### PR TITLE
Add inflection to test app generator to "fix" Atlas paths

### LINKED ISSUE #693: Generator is creating bad routes for atlas
### Descriptive summary

In Master when you create the test app it creates bad routes for Atlas.  If I had to hazard a guess the plural to singular is causing the atla instead of the atlas.  I do not see the same behavior when I generate a class not ending in `s`.

screen shot 2017-04-03 at 2 00 43 pm

### Expected behavior

should get routes for atlas

### Actual behavior

get routes for atla

### Steps to reproduce the behavior

1. run rake engin_cart:generate
1. cd .internal_test_app
1. run rake routes

### Related work

Link to related tickets or prior related work here.

### PR DESCRIPTION
**Pull request by mjgiarlo**
Also, clarified this in the documentation: https://github.com/projecthydra/sufia/wiki/Customizing-your-work-types

Fixes #693

### CODE REVIEW

File: spec/test_app_templates/lib/generators/test_app_generator.rb [hunk 1]
```diff
@@ -10,9 +10,16 @@ def install_engine
   def create_generic_work
     generate 'hyrax:work GenericWork'
   end
-  
+
   def create_atlas_work
     generate 'hyrax:work RareBooks/Atlas'
+    append_file 'config/initializers/inflections.rb' do
+      <<~HEREDOC
```
Comment (cam156) [inline spec/test_app_templates/lib/generators/test_app_generator.rb]: @mjgiarlo for future people look at this code can you add a comment that this was done becuase it created a bad plural?

**Comment by mjgiarlo** [comment]
@cam156 There's only one "broken" set of paths that the test app sets up, and that's for atlases. We can't anticipate every single irregular noun and ship all those inflections. Rails has a pattern for handling this, which we now link to from our documentation: https://github.com/projecthydra/sufia/wiki/Customizing-your-work-types

File: spec/test_app_templates/lib/generators/test_app_generator.rb (hunk 1 above)
Comment (mjgiarlo) [inline spec/test_app_templates/lib/generators/test_app_generator.rb]: Done!

**Comment by atz** [comment]
One half of CI failed, possibly on an intermittent error.  Restarted job.

**Pull request merged**